### ISSUE TITLE
Is there any way to use ssl

### ISSUE THREAD

**Issue by abhi634367** [gnmyt/MCDash#136]

**Comment by marcjoancr** [comment]
Yes, with a reverse proxy, if you use Apache2 then this tut should do it: https://www.cherryservers.com/blog/how-to-install-and-configure-apache-reverse-proxy-server-with-ssl-encryption
Otherwise search for reverse proxy for nginx or the web server solution of your election.
